priv/repo/migrations/20210417003100_drop_last_seen_customer.exs

```diff
@@ -0,0 +1,12 @@
+defmodule ChatApi.Repo.Migrations.DropCusotmerLastSeen do
+  use Ecto.Migration
+  import Ecto.Query
+
+  alias ChatApi.Repo
+
```

Comment:
@utsav0209 mind fixing the warnings above?

Reply:
i'm also thinking maybe we shouldn't drop this column quite yet -- maybe we can handle this migration in a separate PR to give us time to test that everything is working as expected?

Reply:
Yeah, I think that's a good idea, can put PR later for this migration once we are sure things are working as expected. I will remove the migration from this PR in that case

Reply:
@reichert621 I am also removing the required check for the last_seen field for this to move forward